freqtrade/rpc/rpc.py

```diff
@@ -144,13 +144,17 @@ def _rpc_show_config(config, botstate: Union[State, str]) -> Dict[str, Any]:
         }
         return val
 
-    def _rpc_trade_status(self) -> List[Dict[str, Any]]:
+    def _rpc_trade_status(self, trade_ids=None) -> List[Dict[str, Any]]:
```

Comment:
i think we might want to type this with `trade_ids: List[int] = []`  ... (check mypy output after this to see if that causes problems).

Reply:
Thank you for your help. I changed it like you suggested and then, with another change in the `telegram.py` it passed the mypy tests.